doc/akri_connector/connector-metadata-schema.json

```diff
@@ -66,7 +66,7 @@
         },
         "key": {
           "type": "string",
-          "description": "The default broker state store key that will be published to"
+          "description": "The default broker state store key that will be published to. Values are allowed to contain deploy-time parameters like 'dss/{deviceName}/{inboundEndpointName}'. The supported parameter tokens are: {deviceName}, {inboundEndpointName}, {assetName}, {connectorClientId}, {namespace}, and {endpointType}."
```

Comment:
@hansgschossmann you mentioned "namespace" in the meeting yesterday as a potential parameter here. Just to confirm, but you were referring to the Kubernetes namespace that connector is deployed in, correct?